### ISSUE TITLE
windows10,after add layout/index.js,Failed to compile.

### ISSUE THREAD

**Issue by qppq54s** [umijs/umi#78]
my step:
1. umi dev
2. add pages/index.js and pages/list.js
3. add layouts/index.js  and throw error
```
Failed to compile.

./pages/.umi/router.js
Syntax error: C:\Users\qiupe\Desktop\test\umitest\pages\.umi\router.js: Bad character escape sequence (3:49)

  1 | import { Router as DefaultRouter, Route, Switch } from 'react-router-dom';
  2 | import dynamic from 'umi/dynamic';
> 3 | import Layout from 'C:\Users\qiupe\Desktop\test\umitest\layouts\index.js';
    |                                                  ^
  4 |
  5 |
  6 | const Router = window.g_CustomRouter || DefaultRouter;
```
then i change my app name to 'myapp',throws:
```
Failed to compile.

./pages/.umi/router.js
Module not found: Can't resolve 'C:UsersqiupeDesktop    estmyapplayoutsindex.js' in 'C:\Users\qiupe\Desktop\test\myapp\pages\.umi'

```

**Comment by sorrycc** [comment]
It should be the winPath problem, I'll fix it on my windows computer tonight.

**Comment by bigbigbo** [comment]
the same problem

**Comment by bigbigbo** [comment]
path problem also in `<Route exact path="/mime\history\index" component={() => <div>Compiling...</div>}></Route>`

**Comment by sorrycc** [comment]
It's fixed via 4ebf004d26efe7b8cc72a60b78f28c8c0646a029, try umi@1.0.0-rc.2
